aten/src/ATen/native/native_functions.yaml

```diff
@@ -4441,7 +4441,7 @@
   use_c10_dispatcher: full
   variants: method, function
   dispatch:
-    CPU: legacy::cpu::_th_trace
+    CPU: trace
```

Comment:
Let's call this "trace_cpu" to avoid confusion

Reply:
done